src/main/java/gradle_clojure/clojure/tasks/ClojureCompile.java

```diff
@@ -80,10 +82,11 @@ public void setNamespaces(List<String> namespaces) {
     this.namespaces = namespaces;
   }
 
-  @TaskAction
   @Override
+  @TaskAction
   public void compile() {
     getDestinationDir().mkdirs();
+    GFileUtils.cleanDirectory(getDestinationDir());
```

Comment:
I _believe_ this class is internal to Gradle. Usually I judge based on it's presence in [the Javadoc](https://docs.gradle.org/current/javadoc/). I don't think they have all of the internal classes in internal packages.

We could leave it though, as long as we keep testing new versions of Gradle we'll find out if it changes.

Reply:
I just realised I can use `getProject().delete(...)` to handle it using a public API.